src/middlewared/middlewared/plugins/smb_/registry_share.py

```diff
@@ -66,7 +66,7 @@ async def netconf(self, **kwargs):
 
     @private
     async def reg_listshares(self):
```

Comment:
Creating a fresh share raises a `KeyError` wrt to the `comment` key.

``` 
File "/usr/lib/python3.9/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3/dist-packages/middlewared/plugins/smb_/registry_share.py", line 353, in share_to_smbconf
    if data['comment']:
KeyError: 'comment'
```

In the `share_to_smbconf` method, I'd change it to:
```
if data.get('comment', ''):
```